web-app/HeimScripts/heatmap/run.r

```diff
@@ -7,6 +7,8 @@ main <- function(max_rows=50){
   if(ncol(df) > 3){
     variances <- apply(df[,3:ncol(df)],1,var) # Calculating variance per probe
     df["SIGNIFICANCE"] <- variances
+    df["MEAN"] <- apply(df[,3:ncol(df)],1,mean, na.rm = T)
+    df["SD"] <- apply(df[,3:ncol(df)],1,sd, na.rm = T)
```

Comment:
Why not calculate z-score here as well, if mean and sd are calculated here? Then you can remove the mean and sd column again (or actually these two columns do not need to be added in the first place). It will save 2 columns later in the melted data.frame